enterprise/frontend/src/metabase-enterprise/group_managers/components/UserTypeCell/UserTypeCell.styled.tsx

```diff
@@ -10,7 +10,6 @@ export const ChangeTypeButton = styled.button`
 
 export const UserTypeCellRoot = styled.td`
   text-transform: capitalize;
-  text-decoration: underline;
```

Comment:
talked to design and they agreed there shouldn't be underline here.
https://metaboat.slack.com/archives/C064EB1UE5P/p1704386904801489